Lib/encodings/cp1125.py

```diff
@@ -163,8 +163,8 @@ def getregentry():
     0x00f3: 0x0491,     #  CYRILLIC SMALL LETTER GHE WITH UPTURN
     0x00f4: 0x0404,     #  CYRILLIC CAPITAL LETTER UKRAINIAN IE
     0x00f5: 0x0454,     #  CYRILLIC SMALL LETTER UKRAINIAN IE
-    0x00f6: 0x0406,     #  CYRILLIC CAPITAL LETTER BYELORUSSIAN-UKRAINIAN I
-    0x00f7: 0x0456,     #  CYRILLIC SMALL LETTER BYELORUSSIAN-UKRAINIAN I
+    0x00f6: 0x0406,     #  CYRILLIC CAPITAL LETTER BELORUSSIAN-UKRAINIAN I
```

Comment:
These files were generated. And this is still an official name of the character.
```py
>>> import unicodedata
>>> unicodedata.name('\u0406')
'CYRILLIC CAPITAL LETTER BYELORUSSIAN-UKRAINIAN I'
```